src/commonmark/en/content/user/data-administration.md

```diff
@@ -608,6 +610,21 @@ Some aspects of the data synchronization feature to be aware of:
     The validation won't fail for the user with this authority. The
     authority should be assigned to the user, on the central server,
     that will be used for synchronization job.
+    
+  - <div id="skip_changed_before"></div>In specific cases, **the initial 
+    synchronisation of all the data can be redundant.** Such situation can 
+    appear when, for example, a database on the local instance is a fresh 
+    copy of the database present on the central instance, or when it is 
+    preferred to not synchronise old data in favor of initial 
+    synchronisation taking less time. 
+  
+    The *syncSkipSyncForDataChangedBefore* SettingKey can be used to skip 
+    the synchronisation of all the data (data values, Event and Tracker 
+    program data, complete data set registrations) that were *last time 
+    changed before the specified date*. The SettingKey is used in the 
```

Comment:
"last time changed before the specified date" sound a bit awkward to me. I think it is better (from a native English perspective) if you drop the "time"; e.g. "last changed before the specified date"